docs/DeveloperGuide.md

```diff
@@ -382,6 +382,28 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 2. GuestList shows an exit message
 3. GuestList exits the program
 
+**Use case: Get the check-in/out date of a guest**
+
+**MSS**
+1. User search for guest
+2. User request for check-in/out date of the guest
+3. The system show the check-in/out date
+
+**Extensions**
+* 1a. Cannot find the guest.
+  Use case ends.
+  
+**Use case: Edit the email address of guest**
+
+**MSS**
+1. User search for guest
+2. User request for changing the email address
+3. Save the changed address
+
+**Extensions**
+* 1a. Cannot find the guest.
```

Comment:
Should the use case be written like "System returns none as guest does not exist"?